src/api/controller/api/upload.js

```diff
@@ -18,12 +18,12 @@ import {
     createReadStream,
 } from '../../services/fsHelpers';
 
-export const getParser = (type) => {
-    if (!loaders[type]) {
-        throw new Error(`Unsupported document type: ${type}`);
+export const getParser = (parserName) => {
+    console.log('parserName', parserName);
```

Comment:
console.log spotted

Reply:
I don't understand why the ./pre-commit script isn't executed... I think it should send me a warning for this type of mistake